lua/ulx/modules/sh/cfc_crucify.lua

```diff
@@ -0,0 +1,146 @@
+    cross:SetModel( "models/props_c17/gravestone_cross001a.mdl" )
+    cross:SetPos( ply:GetPos() )
+
+    local angles = ply:GetAngles()
+    cross:SetAngles( angles )
+
+    cross:Spawn()
+    cross:Activate()
+
+    local crossPhys = cross:GetPhysicsObject()
+    crossPhys:SetMass( 1000 ) -- make it easier to move the ragdoll attached to the cross
+    crossPhys:EnableMotion( false )
+
+    ply.cross = cross
+    cross.crucifiedPly = ply
+    playerRagdoll.crucifiedPly = ply
+
+    local phys_obj = playerRagdoll:GetPhysicsObjectNum( 1 )
+    local boneID = 2
+
+    while IsValid( phys_obj ) do
+        phys_obj = playerRagdoll:GetPhysicsObjectNum( boneID )
+
+        if phys_obj then
+            phys_obj:SetPos( phys_obj:GetPos() + Vector( 0, 0, 80 ) + angles:Forward() * 10 )
+        end
+        boneID = boneID + 1
+    end
+
+    local rightHandID = playerRagdoll:LookupBone( "ValveBiped.Bip01_R_Hand" )
+    local leftHandID = playerRagdoll:LookupBone( "ValveBiped.Bip01_L_Hand" )
+
+    local rightHandPhys = playerRagdoll:TranslateBoneToPhysBone( rightHandID )
+    local leftHandPhys = playerRagdoll:TranslateBoneToPhysBone( leftHandID )
+
+    constraint.Weld( cross, playerRagdoll, 0, rightHandPhys, 0, false, false )
+    constraint.Weld( cross, playerRagdoll, 0, leftHandPhys, 0, false, false )
+end
+
+function cmd.crucifyPlayers( caller, targets, shouldUncrucify )
+    local affectedPlys = {}
+
+    for _, ply in pairs( targets ) do
+        if not shouldUncrucify then
+            if ulx.getExclusive( ply, caller ) then
+                ULib.tsayError( caller, ulx.getExclusive( ply, caller ), true )
+            elseif not ply:Alive() then
+                ULib.tsayError( caller, ply:Nick() .. " is dead and cannot be crucified!", true )
+            else
+                local err = crucifyPlayer( ply )
+
+                if not err then
+                    table.insert( affectedPlys, ply )
+                else
+                    ULib.tsayError( caller, err, true )
+                end
+            end
+        elseif ply.cross then
+            cmd.uncrucifyPlayer( ply )
+            table.insert( affectedPlys, ply )
+        end
+    end
+
+    if shouldUncrucify then
+        ulx.fancyLogAdmin( caller, "#A uncrucified #T", affectedPlys )
+    else
+        ulx.fancyLogAdmin( caller, "#A crucified #T", affectedPlys )
+    end
+end
+
+local crucifyCommand = ulx.command( CATEGORY_NAME, "ulx crucify", cmd.crucifyPlayers, "!crucify" )
+crucifyCommand:addParam{ type = ULib.cmds.PlayersArg }
+crucifyCommand:addParam{ type = ULib.cmds.BoolArg, invisible = true }
+crucifyCommand:defaultAccess( ULib.ACCESS_ADMIN )
+crucifyCommand:help( "Crucifies target(s)" )
+crucifyCommand:setOpposite( "ulx uncrucify", { _, _, true }, "!uncrucify" )
+
+local function crucifyDisconnectedCheck( ply )
+    if IsValid( ply.ragdoll ) then
+        ply.ragdoll:DisallowDeleting( false )
+        ply.ragdoll:Remove()
+    end
+    if IsValid( ply.cross ) then

```

Comment:
You could use [`SafeRemoveEntity` ](https://wiki.facepunch.com/gmod/Global.SafeRemoveEntity) here